resources/views/clubevent/clubEventView.blade.php

```diff
@@ -276,11 +276,19 @@
 						    <div class="col-md-3 col-sm-2 col-xs-3 no-padding" id="clubStatus{{ $shift->id }}">
 						        @include("partials.shiftStatus")
 						    </div>
-
-						    {{-- Shift USERNAME--}}
-						    <div id="{!! 'userName' . $shift->id !!}" >
-						        {!! $shift->getPerson->prsn_name !!}
-						    </div>
+                            @php
+                            /** @var \Lara\Shift $shift*/
+                            @endphp
+                            @if($shift->getPerson->isNamePrivate() === 0)
+                                {{-- Shift USERNAME--}}
+                                <div id="{!! 'userName' . $shift->id !!}" >
+                                    {!! $shift->getPerson->prsn_name !!}
+                                </div>
+                            @else
+                                <div id="{!! 'userName' . $shift->id !!}" >
+                                    Anonymous
```

Comment:
Ich würde hier eher Status zeigen, z.B. "Mitglied", "Kandidat", "Veteran" und dann mit translate() in allen Sprachen.

Reply:
gute idee